db/migrate/20230316103524_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+
```

Comment:
- [ ] Please make sure to follow the given diagram, as the reference to the users table should be called `author` instead of `user`.